rpc-rs/Cargo.toml

```diff
@@ -14,47 +14,47 @@ edition = "2021"
 exclude = [ "build.rs" ]
 
 [features]
-default = [ "chunkify" ]
+default = [ "otel" ]
 BigInteger = [ "num-bigint" ]
 BigDecimal = [ "bigdecimal" ]
-# non-working feature - used for internal testing of async-rewrite branch of nats-io/nats.rs
-#async_rewrite = [ "nats-experimental" ]
-chunkify = [ "nats" ]
-otel = ["opentelemetry", "tracing-opentelemetry", "nats", "opentelemetry-otlp"]
+prometheus = [ "dep:prometheus" ]
```

Comment:
dep syntax lets you use a feature flag name that is the same name as a dependency crate. In an earlier impementation, the prometheus flag also had a prometheus-exporter crate, but after chatting with Dan I decided to remove the exporter. There's a remaining unimplemented piece which is to push the stats to an endpoint instead of opening a port so prometheus can read them.